pkgs/development/python-modules/inflect/default.nix

```diff
@@ -1,31 +1,32 @@
 {
   lib,
   buildPythonPackage,
-  fetchPypi,
-  isPy27,
+  pythonOlder,
+  fetchFromGitHub,
   more-itertools,
   setuptools-scm,
-  pydantic,
   pytestCheckHook,
   typeguard,
 }:
 
 buildPythonPackage rec {
   pname = "inflect";
-  version = "7.2.1";
-  disabled = isPy27;
-  format = "pyproject";
+  version = "7.3.1";
+  pyproject = true;
 
-  src = fetchPypi {
-    inherit pname version;
-    hash = "sha256-p85eI9Z5hzTyVsGtntUhhrjsJ28QsYzj0+yxnCHrbLY=";
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "jaraco";
+    repo = "inflect";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-J0XgSKPzZIt/7WnMGARXpyYzagBGiqRiuNmNnGKDBrs=";
   };
 
-  nativeBuildInputs = [ setuptools-scm ];
```

Comment:
I guess we also have to add `setuptools` in here. I'll do it.

Reply:
No.

Reply:
=> https://github.com/NixOS/nixpkgs/pull/329849#discussion_r1691063277

Reply:
@dotlambda @natsukium Please advise ^^

Reply:
We should not depend on implicit propagation.

Reply:
What does @mweinelt think?

Reply:
Don't worry, it has been merged already: https://github.com/NixOS/nixpkgs/pull/330114

Reply:
Please stop the pointless discussion.
https://github.com/NixOS/nixpkgs/pull/265993/files#r1385614792

Reply:
I just want to come to a consensus. But given I'm outvoted I'm happy to add setuptools in the future.

Reply:
I don't care.